core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-properties-form/field-properties/values-property/values-property.component.ts

```diff
@@ -23,6 +23,23 @@ export class ValuesPropertyComponent {
         'com.dotcms.contenttype.model.field.ImmutableSelectField'
     ];
 
+    /**
+     * check field is required or not
+     * @readonly
+     * @type {boolean}
+     * @memberof DefaultValuePropertyComponent
+     */
+    get isRequired(): boolean {
+        const validatorFunc = this.group.get(this.property.name).validator;
```

Comment:
@zulqarnainvd I feel like this is the function used before on  [/field-properties/regex-check-property/regex-check-property.component.ts](https://github.com/dotCMS/core/pull/24153/files#diff-bf6d3c20ac772a135861681501d85f592dcd34e647f23e25576652d59416bd42) can we move it to a Utils file and reuse it?

Reply:
Noted